src/app/views/landing/about/about.vue

```diff
@@ -7,8 +7,8 @@
 						class="bolt anim-fade-in-down"
 						src="~img/jolt.svg"
 						alt=""
-						tooltip="This is a lightning bolt."
 						strict-colors
+						v-app-tooltip="$gettext('This is a lightning bolt.')"
```

Comment:
This is the only thing I changed in this file - everything else is auto-formatting.